sdk/cognitivelanguage/azure-ai-language-questionanswering/samples/authoring/sample_export_import_project.py

```diff
@@ -0,0 +1,102 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_export_import_project.py
+
+DESCRIPTION:
+    This sample demonstrates how to export and import a Qna project.
+
+USAGE:
+    python sample_export_import_project.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_QUESTIONANSWERING_ENDPOINT - the endpoint to your QuestionAnswering resource.
+    2) AZURE_QUESTIONANSWERING_KEY - your QuestionAnswering API key.
+"""
+
+from azure.ai.language.questionanswering import projects
+
+
+def sample_export_import_project():
+    # [START export_import_project]
+    import os
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.language.questionanswering.projects import QuestionAnsweringProjectsClient
+
+    # get service secrets
+    endpoint = os.environ["AZURE_QUESTIONANSWERING_ENDPOINT"]
+    key = os.environ["AZURE_QUESTIONANSWERING_KEY"]
+
+    # create client
+    client = QuestionAnsweringProjectsClient(endpoint, AzureKeyCredential(key))
+    with client:
+
+        # create project
+        project_name = "IssacNewton"
+        client.create_project(
+            project_name=project_name,
+            options={
+                "description": "biography of Sir Issac Newton",
+                "language": "en",
+                "multilingualResource": True,
+                "settings": {
+                    "defaultAnswer": "no answer"
+                }
+            })
+
+        # export
+        export_poller = client.begin_export(
+            project_name=project_name,
+            format="json"
+        )
+        export_result = export_poller.result()
+        export_url = export_result["resultUrl"]
```

Comment:
What's the data at this URL? Could we maybe download it, then feed it into the subsequent import?

Reply:
exactly!
this was our first intuition, to use this export url in the import method 
but the thing is, this url is key-protected, and we can only submit public urls for the import method
we thought of including a sample method to download the file form this url
we actually need to ask Rohan from the service team about this